examples/sentence_transformer/unsupervised_learning/query_generation/README.md

```diff
@@ -11,13 +11,13 @@ query: What is Python?
 passage to retrieve: Python is an interpreted, high-level and general-purpose programming language. Python's design philosophy emphasizes code readability with its notable use of significant whitespace. Its language constructs and object-oriented approach aim to help programmers write clear, logical code for small and large-scale projects.
 ```
 
-We showed how to train such models if sufficient training data (query & relevant passage) is available here: [Training MS MARCO dataset](../../training/ms_marco/README.md) 
+We showed how to train such models when sufficient training data (query & relevant passage) is available here: [Training MS MARCO dataset](../../training/ms_marco/README.md) 
 
-In this tutorial, we show to train such models if  **no training data is available**, i.e., if you don't have thousands of labeled query & relevant passage pairs.
+In this tutorial, we show how to train such models if  **no training data is available**, i.e., if you don't have thousands of labeled query & relevant passage pairs.
 
 ## Overview
 
-We use **synthetic query generation** to achieve our goal. We start with the passage from our document collection and create for these possible queries users might ask / might search for.
+We use **synthetic query generation** to achieve our goal.  We start with the passage from our document collection and create possible queries that users might ask or search for.
```

Comment:
```suggestion
We use **synthetic query generation** to achieve our goal. We start with the passage from our document collection and create possible queries that users might ask or search for.
```
One tiny nitpick, but otherwise these are great fixes, thank you!